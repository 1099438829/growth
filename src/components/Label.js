import React from 'react';
import { Text, View } from 'react-native';

const Label = props => (
  <View style={{ paddingLeft: 20, marginTop: 12, marginBottom: -6 }}>
    <Text style={{ fontSize: 14 }}>
      {props.text}
    </Text>
  </View>
);

Label.propTypes = {
  text: React.PropTypes.string.isRequired,
};

export default Label;
