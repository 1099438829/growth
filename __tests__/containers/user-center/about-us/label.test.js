import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../../../../src/containers/user-center/about-us/Label';

describe('test user center / about us / label', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Label text="LabelText" />,
    );

    const treeJson = tree.toJSON();
    expect(treeJson).toMatchSnapshot();
  });
});
