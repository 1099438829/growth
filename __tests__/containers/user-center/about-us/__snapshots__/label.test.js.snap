// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test user center / about us / label renders correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": -6,
      "marginTop": 12,
      "paddingLeft": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 14,
      }
    }
  >
    LabelText
  </Text>
</View>
`;
